src/EditorFeatures/CSharpTest/CodeActions/InlineTemporary/InlineTemporaryTests.cs

```diff
@@ -3756,7 +3756,7 @@ public void M()
 {
     public void M()
     {
-        var s2 = string.Replace($""hello"", ""world"");
+        var s2 = string.Replace((string)$""hello"", ""world"");
```

Comment:
@dotnet/roslyn-ide Could someone look at this? It seems strange to me that we need to generate a new cast just because we're using C# 7.3+, but this looks like the current IDE behavior. Is this a bug or am I forgetting something?

Reply:
That's definitely a bug in something. Whether it's the IDE or compiler who knows...

Reply:
In any case: if you don't investigate that prior to merge, I'd say you should instead disable the test (rather than update to an incorrect baseline) and file a tracking bug. The IDE generally does _not_ do the "assert the wrong thing for now" pattern.